### ISSUE TITLE
Create a Glossary page

### ISSUE THREAD

**Issue by jbogarthyde** [in-toto/in-toto.io#37]
-  [ ] Adapt content of [Terminology](https://github.com/in-toto/docs/blob/v1.0/in-toto-spec.md#17-terminology) to an alphabetized table format. 
- [ ] Add terms as needed.
- [ ] Add links to Glossary on Doc Home Page and in Getting Started menu
- [ ] Link to terms in the glossary from the tech doc where necessary (first time term appears on a page or section)

See [Umbrella Issue](https://github.com/in-toto/docs/issues/85#top)

**Comment by DarikshaAnsari** [comment]
@lukpueh @chalin Hi, I haven't started working on this issue yet. Could you please suggest where I should add this section to our website?

**Comment by lukpueh** [comment]
I don't know. Maybe as a subsection in docs? Between specs and faq?

**Comment by DarikshaAnsari** [comment]
so after adding this terminology section on glossary page i have to remove it from the in-toto-spec.md.

**Comment by DarikshaAnsari** [comment]
So,My task is to create a glossary page containing a terminology table, link this page to the index of our documentation, and ensure that the first occurrence of any term in the documentation is linked to the glossary.

**Comment by lukpueh** [comment]
I don't think we want to remove it from the spec. I suggest to either just make the new glossary page point to the spec or duplicate for now (and make a note in https://github.com/DarikshaAnsari/in-toto-docs/issues/22).

**Comment by DarikshaAnsari** [comment]
> > So,My task is to create a glossary page containing a terminology table, link this page to the index of our documentation, and ensure that the first occurrence of any term in the documentation is linked to the glossary.
> 
> Sounds like a plan. IIUC that would mean that you duplicate the glossary.

That's why I asked you to remove it from in-toto-spec.md.

**Comment by lukpueh** [comment]
> I don't think we want to remove it from the spec. I suggest to ~either just make the new glossary page point to the spec or~ duplicate for now (and make a note in [DarikshaAnsari/in-toto-docs#22](https://github.com/DarikshaAnsari/in-toto-docs/issues/22)).

☝️
